app/views/miq_request/_pre_prov.html.haml

```diff
@@ -42,7 +42,7 @@
           %td
             = h(number_to_human_size(row.allocated_disk_storage))
           %td
-            = h(row.deprecated)
+            = h(row.deprecated ? true : false)
```

Comment:
Could this perhaps be `_("true")` and `_("false")` for the sake of translations?